src/events/render-advert-label.spec.ts

```diff
@@ -61,11 +61,11 @@ describe('shouldRenderLabel', () => {
 		});
 	});
 
-	it('does NOT render an ad label for fluid ads', async () => {
+	it('renders an ad label for fluid ads', async () => {
```

Comment:
Instead of testing for fluid ads, could we check that a label should render for a fabric ad, and shouldn't render for a manual multiple, for example? That would make the test more relevant to the new logic